exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdDistinctRowCount.java

```diff
@@ -155,18 +155,18 @@ private Double getDistinctRowCountInternal(TableScan scan, RelMetadataQuery mq,
     }
 
     double s = 1.0;
-    boolean allCols = true;
+    boolean allColsHaveNDV = true;
```

Comment:
Unrelated to this line but on previous lines 145-148 seem to be duplicates of lines 136-139.

Reply:
can you mark this done if this comment is addressed .. seems like the duplicate lines are there.

Reply:
Done